### ISSUE TITLE
run_tests.sh fails on latest release 1.5.4

### ISSUE THREAD

**Issue by amo13** [ciur/papermerge#272]
With the latest release 1.5.4, the run_tests fails with the following results:
```
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
..........................................................F.................s.s...........................................................................................................................
======================================================================
FAIL: test_folder_validation_against_xss_titles (papermerge.test.test_document.TestDocument)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/amo/.cache/yay/papermerge/src/papermerge-1.5.4/papermerge/test/test_document.py", line 369, in test_folder_validation_against_xss_titles
    folder_a.full_clean()
AssertionError: ValidationError not raised

----------------------------------------------------------------------
Ran 202 tests in 13.503s

FAILED (failures=1, skipped=2)
Destroying test database for alias 'default'...
```

Maybe this (`test_folder_validation_against_xss_titles`) has something to do with #271?

**Comment by ciur** [comment]
@amo13, the failed test is related to #228. I fixed #228 by escaping user input (correct mitigation of XSS) instead of validating user input with custom filters (latter is checked by failing test). Because custom filters validation "feature" was replaced with escape that tests is absolute and should be removed anyway.

Let me put in other words: failing test was removed immediately AFTER release 1.5.4 was tagged. Correct it would be to remote failing test BEFORE tagging release 1.5.4.
I am not sure how to mitigate this situation.
